packages/src/view/base.ts

```diff
@@ -152,18 +161,16 @@ export class BaseElementComponent<T extends Element = Element, K extends Angular
     }
 
     updateWeakMap() {
+        this.KEY_TO_ELEMENT?.set(this.key, this.nativeElement)
         NODE_TO_ELEMENT.set(this.element, this.nativeElement);
         ELEMENT_TO_NODE.set(this.nativeElement, this.element);
-        ELEMENT_TO_COMPONENT.set(this.element, this);
```

Comment:
ELEMENT_TO_COMPONENT is a very important feature for implementing complex editor requirements.